addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8951,12 +8960,12 @@ msgstr ""
 
 #: xbmc/epg/Epg.cpp
 msgctxt "#19507"
-msgid "Serious/Classical/Religious/Historical Movie/Drama"
+msgid "Serious/Classical/Religious/Historical movie/drama"
```

Comment:
I dont mind and for the record I totally agree with the drawing and quartering of the guilty parties.

I will start adding spaces between all ```a \ b``` now as 3 people including myself agree on that.

All I can say I thank good I dont use PVR. :) hahahaha Maybe if I had the need I would.